common/shlibs

```diff
@@ -2741,7 +2741,7 @@ libgspell-1.so.2 gspell-1.8.0_1
 libotf.so.1 libotf-0.9.16_1
 libimagequant.so.0 libimagequant-2.8.2_1
 libDOtherSide.so.0 dotherside-0.5.2_1
-libstlink-shared.so.1 stlink-1.3.0_1
+libstlink.so.1 stlink-1.5.1_1
```

Comment:
I think it is better to remove this entry since nothing depends on it.

Reply:
With the discussion on #2750 it has been sorta decided to keep shlibs for packages that still exist on the repo.

Sorry for the incovenience